presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -125,6 +125,7 @@
     public static final String QUERY_MAX_TOTAL_MEMORY_PER_NODE = "query_max_total_memory_per_node";
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_ROW_COUNT = "dynamic_filtering_max_per_driver_row_count";
     public static final String DYNAMIC_FILTERING_MAX_PER_DRIVER_SIZE = "dynamic_filtering_max_per_driver_size";
+    public static final String DYNAMIC_FILTERING_PROBE_SIDE_BLOCKING_TIMEOUT = "dynamic_filtering_probe_side_blocking_timeout";
```

Comment:
I would simplify this property and just make it:
```
dynamic_filtering_blocking_timeout
```
now that we have partition pruning on coordinator. Note that lazy dynamic filters on coordinator should be separate PR (and not this one).